code/modules/mob/living/carbon/xenomorph/hive_datum.dm

```diff
@@ -69,6 +69,8 @@
 /datum/hive_status/proc/can_hive_have_a_queen()
 	return get_total_xeno_number() < xenos_per_queen
 
+/datum/hive_status/normal/can_hive_have_a_queen()
+	return get_total_xeno_number() + stored_larva < xenos_per_queen
```

Comment:
just override get_total_xeno_number()

Reply:
Check the previous change requests.
`get_total_xeno_number()` is used on a different proc.